test-data/unit/check-plugin-attrs.test

```diff
@@ -31,7 +31,8 @@ class A:
 reveal_type(A)  # N: Revealed type is "def (a: builtins.int, b: builtins.list[builtins.int], c: builtins.str =, d: builtins.int =) -> __main__.A"
 A(1, [2])
 A(1, [2], '3', 4)
-A(1, 2, 3, 4)  # E: Argument 2 to "A" has incompatible type "int"; expected "list[int]" # E: Argument 3 to "A" has incompatible type "int"; expected "str"
+A(1, 2, 3, 4)  # E: Argument 2 to "A" has incompatible type "int"; expected "List[int]" \
```

Comment:
These tests are failing after unclean merge - a PR was recently merged where 3.8-style legacy typing aliases were finally disabled. You can run `pytest mypy/test/testcheck.py::TypeCheckSuite::check-plugin-attrs.test --update-data` or just retry the merge.